source/MaterialXTest/GenShaderUtil.cpp

```diff
@@ -341,7 +314,7 @@ void testUniqueNames(mx::GenContext& context, const std::string& stage)
 }
 
 bool generateCode(mx::GenContext& context, const std::string& shaderName, mx::TypedElementPtr element,
-                  std::ostream& log, std::vector<std::string>testStages)
+                  std::ostream& log, std::vector<std::string>testStages, std::vector<std::string>& sourceCode)
```

Comment:
Could use mx::StringVec here, and testStages could be passed as a const reference.

Reply:
Yes. Been remiss in using those out of habit.